### PR TITLE
Added tables to dq results pages

### PR DESCRIPTION
**Pull request by maxtrevor**
This MR adds some tables of information to the DQ results pages, as requested by LVK reviewers

### CODE REVIEW

File: bin/plotting/pycbc_page_template_bin_table [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+trigger_rate = [n / livetime for n in num_triggers]
+
+col_names = ['Template Bin', 'Number of Templates', '% of Templates',
+             'Number of Loud Triggers', '% of Loud Triggers',
+             'Loud Trigger Rate (Hz)']
+columns = [bin_names, num_templates, frac_templates,
+           num_triggers, frac_triggers, trigger_rate]
+columns = [np.array(c) for c in columns]
+
+html_table = pycbc.results.html_table(columns, col_names,
```
Comment (tdent) [inline bin/plotting/pycbc_page_template_bin_table:59]: Minor point: this seems to default to 3 decimal places on all floats.  This is too many s.f. for times in seconds (1,290.199 s?) but not enough for fractions of templates or triggers (% of templates displayed as 0 in one case).

**Review by tdent — CHANGES_REQUESTED** [review]
It seems to work, so approve generally.  But a couple of points to address on percentages and 'loud triggers'

File: bin/plotting/pycbc_page_template_bin_table [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+bin_names.append('Total')
+num_triggers.append(total_triggers)
+num_templates.append(total_templates)
+frac_triggers = [n / total_triggers for n in num_triggers]
+frac_templates = [n / total_templates for n in num_templates]
+trigger_rate = [n / livetime for n in num_triggers]
+
+col_names = ['Template Bin', 'Number of Templates', '% of Templates',
+             'Number of Loud Triggers', '% of Loud Triggers',
```
Comment (tdent) [inline bin/plotting/pycbc_page_template_bin_table:53]: as noted via chat, better to have a quantitative definition of 'Loud' visible somewhere

File: bin/plotting/pycbc_page_template_bin_table (hunk 1 above)
Comment (maxtrevor) [reply bin/plotting/pycbc_page_template_bin_table:59]: Any idea how to change the precision? I agree that its currently not ideal.

Comment (tdent) [reply bin/plotting/pycbc_page_template_bin_table:59]: Nothing beyond what you can see here .. https://pycbc.org/pycbc/latest/html/_modules/pycbc/results/table_utils.html#html_table

File: bin/plotting/pycbc_page_dq_table [hunk 3]
```diff
@@ -39,10 +39,13 @@ frac_livetimes = [lt / total_livetime for lt in livetimes]
 state_names = list(dq_states.values()) + ['Total']
 columns = [state_names, livetimes, frac_livetimes]
 columns = [np.array(c) for c in columns]
-col_names = ['DQ State', 'Livetime', 'Fractional Livetime']
+col_names = ['DQ State', 'Livetime', '% of Livetime']
+
+format_strings = [None, '#.#', '#.##%']
```
Comment (tdent) [inline bin/plotting/pycbc_page_dq_table]: If I understand correctly the '%' here automatically converts a fraction into a percentage (and prints '%' after it).

File: bin/plotting/pycbc_page_template_bin_table [hunk 4]
```diff
@@ -56,11 +59,16 @@ columns = [bin_names, num_templates, frac_templates,
            num_triggers, frac_triggers, trigger_rate]
 columns = [np.array(c) for c in columns]
 
+format_strings = [None, '#', '#.###%', '#', '#.###%', '#.##E0']
```
Comment (tdent) [inline bin/plotting/pycbc_page_template_bin_table]: Seems OK, although I think one decimal place for '% of loud triggers' should be sufficient (ie `'#.#%'`)

**Review by tdent — APPROVED** [review]
Approve modulo maybe reducing the number of d.p. on % of loud triggers

**Pull request merged**